### PR TITLE
Ports from Strawberry Jam into Communal Helper

### PR DESCRIPTION
**Pull request by catapillie**
We're been asked to port [Strawberry Jam](https://gamebanana.com/mods/424541) entities into Communal Helper for maintenance and potential updates, bugfixes, etc...

Notes and important stuff:
* Entities ported from SJ must have their name changed. This is the pattern used: `CommunalHelper/SJ/...`.
* Graphics will be put under `Graphics/Atlases/Gameplay/objects/CommunalHelper/strawberryJam/`.
* Sprite data (`.xml`) can be pasted `Graphics/CommunalHelper/Sprites.xml` for now. (Should it be changed to a separate .xml?)
* Lönn plugins and Ahorn plugins will be put under `Loenn/entities/strawberry_jam` and `Ahorn/entities/strawberryJam` respectively.
* Mark Strawberry Jam en_gb.lang entries with `[Strawberry Jam]` for clarity. For Lönn plugins, we'll append `[Strawberry Jam]` in placement names, and in ahorn plugins, we'll write `(Strawberry Jam) (Communal Helper)`.

TODO:

* **Air Time Music Controller**
    - [x] Port implementation from `AirTimeMusicController.cs`
    - [x] Write Lönn plugin
    - [x] Copy Ahorn plugin
* **Bullet Time Controller**
    - [x] Port implementation from `BulletTimeController.cs`
    - [x] Write Lönn plugin
    - [x] Copy Ahorn plugin
* **Expiring Dash Refill**
    - [x] Port implementation from `ExpiringDashRefill.cs`
    - [x] Write Lönn plugin
    - [x] Copy Ahorn plugin
* ~~**Dash Count Trigger**~~ _(cancelled port)_
* **Show Hitbox Trigger**
    - [x] Port implementation from `ShowHitboxTrigger.cs`
    - [x] Write Lönn plugin
    - [x] Copy Ahorn plugin
* ~~**Switch Crate**~~ _(moved to [Batteries](https://github.com/CommunalHelper/Batteries/issues/3))_
* ~~**Switch Crate Holder**~~ _(moved to [Batteries](https://github.com/CommunalHelper/Batteries/issues/3))_
* ~~**Switch Door**~~ _(moved to [Batteries](https://github.com/CommunalHelper/Batteries/issues/3))_
* ~~**UFO**~~ _(planned for another update, plugins staying hidden for now)_
* **Grab Temple Gate**
    - [x] Port implementation from `GrabTempleGate.cs`
    - [x] Write Lönn plugin
    - [x] Copy Ahorn plugin
* **Loop Block**
    - [x] Port implementation from `LoopBlock.cs`
    - [x] Write Lönn plugin
    - [x] Copy Ahorn plugin
    - [x] Add missing SFX
* **Solar Elevator**
    - [x] Port implementation from `SolarElevator.cs`
    - [x] Write Lönn plugin
    - [x] Copy Ahorn plugin
    - [x] Change default dialog and sfx entries (in entity option)
    - [x] Add missing SFXs
* ~~**Dash Zip Mover**~~ _(may be ported another way, likely as an entity option to already an existing entity in Communal Helper)_
* **Photosensitive Flag Controller**
    - [x] Port implementation from `PhotosensitiveFlagController.cs`
    - [x] Write Lönn plugin
    - [x] Copy Ahorn plugin
* **Exploding Strawberry**
    - [x] Port implementation from `ExplodingStrawberry.cs` and `CustomEntitySprites.xml`
    - [x] Write Lönn plugin
    - [x] Copy Ahorn plugin
* **Wormhole Booster**
    - [x] Port implementation from `WormholeBooster.cs` and `CustomEntitySprites.xml`
    - [x] Write Lönn plugin
    - [x] Copy Ahorn plugin
    
Other things:
* [x] Copy SJ's SFX bank events into CommunalHelper's (renaming some events)

### CODE REVIEW

**Comment by l-Luna** [comment]
minor `ShowHitboxTrigger` bugfix: it no longer stops sound sources from being displayed while the player has the debug menu open inside a show hitbox trigger

**Comment by l-Luna** [comment]
Switch Door relies on batteries' Battery Gate, and Switch Crate stuff should probably be implemented as an improvement to Batteries instead.

**Comment by catapillie** [comment]
> Switch Door relies on batteries' Battery Gate, and Switch Crate stuff should probably be implemented as an improvement to Batteries instead.

Moving those to [Batteries](https://github.com/CommunalHelper/Batteries) since they are under the Communal Helper organization. Ticking them off in this PR.

**Comment by catapillie** [comment]
Also going to skip porting Dash Zip Movers. They can be ported as a new (better) mechanic or as an option to Connected Zip Movers.

File: Ahorn/entities/strawberryJam/UFO.jl [hunk 1]
```diff
@@ -0,0 +1,38 @@
+﻿module SJ2021UFO
```
Comment (catapillie) [inline Ahorn/entities/strawberryJam/UFO.jl]:
```suggestion
CommunalHelperUFO
```

**Comment by catapillie** [comment]
Going to hide the UFO plugins as they're being worked on for a future update.

**Comment by catapillie** [comment]
Cancelling the Dash Count Trigger port (Kahuna says it's not a good idea, because bad/cursed implementation or whatever)

**Pull request merged**